packages/itwin/property-grid/README.md

```diff
@@ -201,32 +221,39 @@ Adding a custom settings menu item can be done as follows:
 
 Define a menu item component:
 
+<!-- [[include: [PropertyGrid.ExampleSettingsMenuItemImports, PropertyGrid.ExampleSettingsMenuItem], tsx]] -->
+<!-- BEGIN EXTRACTION -->
+
 ```tsx
+import { PropertyGridSettingsMenuItem } from "@itwin/property-grid-react";
+
 function ExampleSettingsMenuItem() {
   return (
     // render using `PropertyGridSettingsMenuItem` to get consistent style
-    <PropertyGridSettingsMenuItem
-      id="example"
-      onClick={() => {
-        console.log("Settings menu clicked!");
-      }}
-    >
+    <PropertyGridSettingsMenuItem id="example" onClick={() => {}}>
```

Comment:
Can you update `onClick` handler to something like:

```ts
onClick={() => {
  // handle settings item clicked
}}
```

Reply:
Changed